src/lib/pmd/PmdCatalogWrapper.ts

```diff
@@ -168,7 +168,7 @@ export class PmdCatalogWrapper extends PmdSupport {
 			this.outputProcessor.processOutput(stdout);
 			this.logger.trace(`monitorChildProcess has received exit code ${code}`);
 			if (code === 0) {
-				res([!!code, stdout]);
```

Comment:
The return code as a boolean lost it's significance since the xml needs to be processed if it's a 0 or a 4. See changes below.